app/src/main/java/com/quran/labs/androidquran/widgets/QuranPageLayout.java

```diff
@@ -141,16 +143,26 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
     }
   }
 
+  private int getBorderWidth(Drawable drawable) {
+    if (drawable == lineDrawable)
+      return lineDrawable.getIntrinsicWidth();
+
+    // convert 2mm to pixels
+    return (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_MM,
+        2f,
```

Comment:
What's the rationale for using exactly 2mm? Why not 1mm for example?

Reply:
It seems to match the official screenshots https://android.quran.com/images/scr_2.png .
Smaller wouldn't achieve the desired effect (i.e let the brain know which side you're on).

To answer @ahmedre 's question I wasn't aware that dp was a resolution independent size so can replace mm with the equivalent dp if that's preferred/more efficient.